Bottom: 0c63057f0a8690dbf0266b6cb18fc49d0f488d2a
Top:    d3fadec15ee3b6ce85491bbc56c40576a4e30262
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-02-11 00:43:15 +0000

Add warning about bogus code in strflags.c


---

diff --git a/src/strflags.c b/src/strflags.c
index 3e57696..d6d6273 100644
--- a/src/strflags.c
+++ b/src/strflags.c
@@ -576,6 +576,7 @@ dump_flag (FlagType * f)
 {
   int l;
   printf ("F:%08x T:[", f->f);
+#warning THIS NEXT LINE IS BOGUS, THE ARRAY IS HALF THIS LENGTH
   for (l = 0; l < (MAX_LAYER + 7) / 8; l++)
     printf (" %02x", f->t[l]);
   printf ("]");
